LibreHardwareMonitorLib/Interop/AtiAdlxx.cs

```diff
@@ -130,7 +130,19 @@ internal static class AtiAdlxx
         public static extern ADLStatus ADL2_Adapter_FrameMetrics_Start(IntPtr context, int adapterIndex, int displayIndex);
 
         [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
-        public static extern ADLStatus ADL2_Adapter_FrameMetrics_Stop(IntPtr context, int adapterIndex, int displayIndex);       
+        public static extern ADLStatus ADL2_Adapter_FrameMetrics_Stop(IntPtr context, int adapterIndex, int displayIndex);
+
+        public static bool ADL_Method_Exists(string ADL_Method)
+        {
+            try
+            {
+                return Kernel32.GetProcAddress(Kernel32.LoadLibrary(DllName), ADL_Method) != IntPtr.Zero;
```

Comment:
This should not throw and LoadLibrary calls should also be freed with FreeLibrary.

Reply:
Changed the function to add FreeLibrary